debian/bionic/foreman/control

```diff
@@ -54,15 +54,15 @@ Description: metapackage providing PostgreSQL dependencies for Foreman
  .
  The foreman package should be installed along with this package.
 
-Package: foreman-sqlite3
+Package: foreman-nulldb
```

Comment:
I feel that we should mirror these subpackage names between RPM and debs. In RPM we have `foreman-build`. Would it make sense to do the same on debs?

Reply:
Sure :heavy_check_mark:

Reply:
@theforeman/packaging this is correct, right?